hack/graph-util.py

```diff
@@ -11,17 +11,9 @@
 
 import yaml
 
-
-try:
-    from builtins import FileExistsError  # Python 3
-except ImportError:
-    FileExistsError = OSError  # sloppy hack for Python 2
-
-try:
-    from urllib.request import Request, urlopen  # Python 3
-except ImportError:
-    from urllib2 import Request, urlopen  # Python 2
```

Comment:
Do we need to drop Python 2 support?  Looks like [they have `Pool` too][1], it's just not a context manager:

```console
$ python2
Python 2.7.5 (default, May 31 2018, 09:41:32) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-28)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import multiprocessing
>>> with multiprocessing.Pool(processes=16) as pool:
...     print(pool)
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: __exit__
```

[1]: https://docs.python.org/2/library/multiprocessing.html#multiprocessing.pool.multiprocessing.Pool

Reply:
ah, maybe [`starmap`][1] is new in 3.3.

[1]: https://docs.python.org/3/library/multiprocessing.html#multiprocessing.pool.Pool.starmap

Reply:
but we may be fine with 2.7's `map` and not need `starmap`...

Reply:
I tried to use a closure, and those don't pickle and so won't work with map. I'll try with a lambda and see how if works.

Reply:
```
AttributeError: Can't pickle local object 'push.<locals>.<lambda>'
```